framework/ramcache.py

```diff
@@ -176,7 +176,7 @@ def invalidate(cls):
   @classmethod
   def check_for_distributed_invalidation(cls):
     """Check if any appengine instance has invlidated the cache."""
-    singleton = cls.get(cls.SINGLETON_KEY, read_policy=db.STRONG_CONSISTENCY)
+    singleton = cls.SINGLETON_KEY.get()
```

Comment:
`read_policy` has been deprecated is, now, a synonymn of `read_consistency`. Although the [document](https://googleapis.dev/python/python-ndb/latest/query.html?highlight=strong#google.cloud.ndb.query.Query.__iter__) suggests that we may use `read_consistency=ndb.STRONG` for strong consistency, I get the following error indicating that this library does not have any such constant.

> Traceback (most recent call last):
  File "/mnt/c/Users/Shivam/ubuntu/chromium-dashboard/lib/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/mnt/c/Users/Shivam/ubuntu/chromium-dashboard/lib/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/mnt/c/Users/Shivam/ubuntu/chromium-dashboard/lib/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/mnt/c/Users/Shivam/ubuntu/chromium-dashboard/lib/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/mnt/c/Users/Shivam/ubuntu/chromium-dashboard/lib/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/mnt/c/Users/Shivam/ubuntu/chromium-dashboard/lib/flask/views.py", line 89, in view
    return self.dispatch_request(*args, **kwargs)
  File "/mnt/c/Users/Shivam/ubuntu/chromium-dashboard/lib/flask/views.py", line 163, in dispatch_request
    return meth(*args, **kwargs)
  File "/mnt/c/Users/Shivam/ubuntu/chromium-dashboard/framework/basehandlers.py", line 325, in get
    ramcache.check_for_distributed_invalidation()
  File "/mnt/c/Users/Shivam/ubuntu/chromium-dashboard/framework/ramcache.py", line 191, in check_for_distributed_invalidation
    SharedInvalidate.check_for_distributed_invalidation()
  File "/mnt/c/Users/Shivam/ubuntu/chromium-dashboard/framework/ramcache.py", line 179, in check_for_distributed_invalidation
    singleton = cls.SINGLETON_KEY.get(read_consistence=ndb.STRONG)
AttributeError: 'module' object has no attribute 'STRONG'

Reply:
I think that a strongly consistent read can be achieved by using an ancestor query instead of get().  It would look something like:
singleton = cls.query(ancestor=PARENT_KEY).fetch(1)[0]
I have not tried that in practice, so we will need to do some manual testing.

Reply:
okay. For the record, I see that you have implemented this in #1356

Reply:
Yep.  That has merged.   I will try deploying a version of the app with that change later today to tomorrow.  If any problems show up due to that specific change, we will know it independently of the larger change.

Reply:
FYI, my change was deployed yesterday.  There are no obvious problems with it so far.

Reply:
So, you just need to merge main into your branch.

Reply:
Update:- See [this](https://github.com/GoogleChrome/chromium-dashboard/pull/1352#discussion_r650489594) link

Reply:
The linked conversation was resolved.